m4/libevent_openssl.m4

```diff
@@ -3,6 +3,22 @@ dnl OpenSSL support
 AC_DEFUN([LIBEVENT_OPENSSL], [
 AC_REQUIRE([NTP_PKG_CONFIG])dnl
 
+case "$host_os" in
+    darwin*)
+    dnl when compiling for Darwin, attempt to find OpenSSL using brew.
+    dnl We prepend the location given by brew to PKG_CONFIG_PATH path
+    dnl and then export it, so that it can be used in detection below.
+    AC_CHECK_PROG([BREW],brew, brew)
+    if test x$BREW = xbrew; then
+        openssl_prefix=$($BREW --prefix openssl 2>/dev/null)
+        if test xopenssl_prefix != x; then
+            PKG_CONFIG_PATH="$openssl_prefix/lib/pkgconfig:$PKG_CONFIG_PATH"
```

Comment:
Should append not prepend, since otherwise user will not allow to overwrite this

Reply:
also might want to avoid adding a `:` if `PKG_CONFIG_PATH` is empty or not set yet

Reply:
Indeed